CombineUnloadAIDriver.lua

```diff
@@ -740,7 +741,7 @@ end
 ---@return number, number x and z offset of the pipe's end from the combine's root node in the Giants coordinate system
 ---(x > 0 left, z > 0 forward) corrected with the manual offset settings
 function CombineUnloadAIDriver:getPipeOffset(combine)
-	return combine.cp.driver:getPipeOffset(-self.vehicle.cp.combineOffset, self.vehicle.cp.tipperOffset)
+	return combine.cp.driver:getPipeOffset(-self.settings.combineOffsetX:get(), self.settings.combineOffsetZ:get())
```

Comment:
Not sure how exactly this works now but this would probably be a good time to flip the sign of the offset settings. Now that it is called X, it should have the same sign as the engine's x coordinate (left and forward positive). 

If it is the opposite, then it should be called combineOffsetRight or combineRightOffset, declaring that if the number is positive, the offset is to the right.

Reply:
The text on the HUD shows left/right/front/back now.
But I will check on Combine with the pipe on the right side, if it gets flipped.

Reply:
@schwiti6190 yes, they are correct.

Reply:
I was talking about the naming of the variables only, especially the side offset, calling it RightOffset would make clear that right is > 0 (contrary to the X coordinate in the Giants engine, which is > 0 to the left)